clang/include/clang/Basic/arm_sme.td

```diff
@@ -298,3 +298,48 @@ multiclass ZAAddSub<string n_suffix> {
 
 defm SVADD : ZAAddSub<"add">;
 defm SVSUB : ZAAddSub<"sub">;
+
+// SME2 - MOVA
+
+//
+// 2 and 4 vector-group read/write intrinsics.
```

Comment:
Opinion: Given you've defined write and read separately you should have separate comments for read and write